frontend/src/components/pipelines/PipelineRow.vue

```diff
@@ -85,7 +90,8 @@ export default {
     data () {
         const pipeline = this.pipeline
         return {
-            editing: false,
+            editingName: false,
+            pipelineName: pipeline.name,
```

Comment:
Could this go under `input` as per our usual pattern? It threw me slightly that pipeline was here twice.
```suggestion
            input: {
                pipelineName: pipeline.name,
            },
```